lighthouse-cli/test/smokehouse/frontends/smokehouse-bin.js

```diff
@@ -13,13 +14,12 @@
 /* eslint-disable no-console */
 
 const path = require('path');
+const cloneDeep = require('lodash.clonedeep');
 const yargs = require('yargs');
 const log = require('lighthouse-logger');
-
 const {runSmokehouse} = require('../smokehouse.js');
-const {server, serverForOffline} = require('../../fixtures/static-server.js');
 
-const coreTestDefnsPath = `${__dirname}/../test-definitions/core-tests.js`;
+const coreTestDefnsPath = path.join(__dirname, '../test-definitions/core-tests.js');
```

Comment:
I'm confused, this change seems out of character for you :)

Reply:
> I'm confused, this change seems out of character for you :)

I actually laughed out loud when I saw you wrote that line in #12415 and I was very sorry to change it :D

It's the `testDefnPath === coreTestDefnsPath` comparison that trips it up, since it ends up being `lighthouse-cli/test/smokehouse/test-definitions/core-tests.js` compared to `lighthouse-cli/test/smokehouse/frontends/../test-definitions/core-tests.js`, so it needs to be normalized, and this was the easiest way 😿

Reply:
*Patrick adds another example to the (admittedly short) "do you know when it will matter?" list 😉*